### DISCUSSION TITLE
Example usage of syntax highlighting override

### DISCUSSION THREAD

**Discussion by ghost** [willfaught/paige#81] [Q&A] [answered]
[This section](https://github.com/willfaught/paige?tab=readme-ov-file#override) in the README mentions that the syntax highlighting style can be overridden.

Are these overrides along the lines of [this](https://github.com/joeroe/risotto?tab=readme-ov-file#colour-schemes) ?

Can someone please share an example of what a modified `dark.css` looks like ?

**Comment by willfaught** [comment]
I updated the README to point to https://gohugo.io/content-management/syntax-highlighting/#generate-syntax-highlighter-css. Run that command with your preferred style name, then put the output in the appropriate light/dark CSS file.

This is how the default styles were generated (the vscode themes, if I remember correctly).

**Reply by willfaught** [reply]
Correction: The default themes are slightly-modified GitHub dark/light themes.

**Comment by ghost** [comment]
It looks like there's just one style - "xcode" in the [Chroma style gallery](https://xyproto.github.io/splash/docs/all.html).

Edit: Is [this](https://github.com/alecthomas/chroma/tree/master/styles) the list of all available styles?

Edit 2: I think it is.

**Reply by willfaught** [reply, marked as answer]
It used to show all the themes. You can still find them in the Wayback Machine: https://web.archive.org/web/20240225135545/https://xyproto.github.io/splash/docs/all.html
